### ISSUE TITLE
"Write a Review" doesn't work in SKStoreProductViewController

### ISSUE THREAD

**Issue by JagCesar** [UrbanApps/Armchair#46]
Hey,

When I tap "Write a Review" in `SKStoreProductViewController` nothing happens. [Seems I'm not the only one having this issue](https://forums.developer.apple.com/thread/11364). Is this something you're aware of?

![screen shot 2016-04-20 at 11 04 40](https://cloud.githubusercontent.com/assets/923548/14669107/b9eeb27e-06e7-11e6-9286-4f09f7ef6f2a.png)

**Comment by deepmode** [comment]
I have the same problem too when tapping on the "Write a review", nothing happen.

**Comment by coneybeare** [comment]
Are you on a beta build, a development app or something else not official?

**Comment by deepmode** [comment]
It happened in the release built. 
To work around,  I set this Armchair.opensInStoreKit(false)  to make it working.

**Comment by coneybeare** [comment]
yes release build, but if your device is running a beta version of the OS, apple will not let you review.

**Comment by deepmode** [comment]
My device is not running the beta version of iOS. Tested on several iOS device, same thing happened. :(

**Comment by coneybeare** [comment]
ok, then it looks like the bug has not been fixed by Apple yet. I wasn't aware there was an issue, so thanks for pointing that out.

**Comment by jlaws** [comment]
Thanks @deepmode that fixed it for me.  Before I added the Armchair.opensInStoreKit(false) all my users got was a grey screen...
